### ISSUE TITLE
Could not initiate token request when configuring wellknown endpoint for an external identity provider

### ISSUE THREAD

**Issue by piraveena** [asgardeo/asgardeo-auth-spa-sdk#29]
**Describe the issue:**

When we add below config,
```
signInRedirectURL: "http://localhost:8080",
signOutRedirectURL: "http://localhost:8080",
clientID: "9tkW4V9lz9bglg4bExmc5UTzBxUa",
serverOrigin: "https://accounts.asgardeo.io/t/bifrost",
wellKnownEndpoint: "/.well-known/openid-configuration"
```
Authorization code flow is working as expected. But token flow is not working. there is no call to token endpoint in the network trace as well.

**AuthorizationRequest:**
`https://<idp_issuer_value>/authorize?response_type=code&client_id=<clientId>&scope=openid&redirect_uri=https%3A%2F%2Flocalhost%3A4200&response_mode=query&code_challenge_method=S256&code_challenge=-BAT5kFeMw6Mx9ZLZQf5xaW7W320eaiJqWbFJJHm9L0`

**Response:**
`https://localhost:8080/?code=5LjrbgCq8rce41wg`

**Comment by piraveena** [comment]
The root cause for this issue is, SDK depends on the session_state param to initiate the token request.

**Comment by Nirothipan** [comment]
@thivi Is there any possibility to fix this any time soon

**Comment by thivi** [comment]
Resolved by #35 and #36
